angular.json

```diff
@@ -3,7 +3,7 @@
   "version": 1,
   "newProjectRoot": "projects",
   "projects": {
-    "ironcore-getting-started-angular": {
```

Comment:
Is this branched off of the most up to date version? I thought I made these changes before. Maybe I missed them and am mis-remembering.

Reply:
this is what was in master.